src/functions/durableEntity.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpRequest, InvocationContext } from '@azure/functions';
+import * as df from 'durable-functions';
+import { DurableOrchestrationClient, IEntityFunctionContext } from 'durable-functions/lib/src/classes';
+
+df.httpClient(
```

Comment:
I would prefer if durable started the same as all other triggers and accepted an options object instead of the handler directly. It's consistent and the most backwards compatible (you never know what other options you may want to support). We can consider a handler-only pattern _in addition_ to that, but personally I don't think it's worth the extra maintenance on our side.

Reply:
I do have both 😛 In addition to the simpler, more streamlined `df.activity()` and `df.httpClient()`, I also added `df.Client()`, `df.clientComplex()`, and `df.activityComplex()`. It doesn't make sense to add those for orchestrations and entities because they don't accept extra inputs or outputs

Reply:
The options object isn't just for extra inputs and outputs though. It could be for any possible option and it's pretty standard practice to use it from the beginning because you don't know what options you will support moving forward.

The `clientComplex` pattern doesn't match what we already have going for http triggers which also supports both. Regardless - I don't think it's worth supporting both from a testing/maintenance perspective for durable.

Reply:
How does the `clientComplex` pattern not match what we have for HTTP triggers? 

>  I don't think it's worth supporting both from a testing/maintenance perspective for durable.

It's not that much more cost. One is just a wrapper over the other

Reply:
There is no `httpComplex`, so that's why it doesn't match.

When you say "wrapper over the other", that's just the source code part of it. We also have to consider the cost of testing (automated and manual), documentation, bloat in the types files, and the multiplicative effect if we allowed every trigger to do this.

One reason I'm strict about this kind of thing is we can't remove it after the fact - we would have to support it forever to stay backwards compatible. On the flip side, it's super easy to add at any point if we get feedback asking for it.

Reply:
I think I resolved this one too soon when I saw `clientComplex` was removed. The first part of the conversation was about accepting an options object instead of a handler directly and that has not been resolved yet. This has to do with backwards compatibility which is why I feel strongly about it

Reply:
Discussed offline. Agreed to add an overload to accept both an options object and a handler directly. Updated the PR with this decision